### ISSUE TITLE
Remove dead links

### ISSUE THREAD

**Issue by jarikmarwede** [ripienaar/free-for-dev#909]
While I was going through some of the links on this repository I noticed that a few of them didn't work.
Because I was curious I ran [this](https://aignes.com/deadlink.htm) dead link checker on the repository and found a few links that can probably be removed:
* https://bip.io/
* https://atlas.hashicorp.com/boxes/search redirects to a 404 page on app.terraform.io
* https://getclef.com/
* https://fieldbook.com/
* http://www.ironsrc.com/data-flow-management/ times out
* https://www.versioneye.com/ times out
* http://pagodabox.io/ times out
* http://java-monitor.com/forum/welcome.php/ times out
* http://visualops.io/ times out
* https://jetbrains.com/youtrack/buy/open_source_incloud.jsp redirects to a 404 page. Second link to it still works (https://www.jetbrains.com/youtrack/buy)

It would be good if someone could double-check them to see if they just don't work for me.

**Comment by ripienaar** [comment]
yeah this all looks as you say to me, we can kill all - except update the last url of course

**Comment by jarikmarwede** [comment]
Ok, I created a few pull requests for it.
I don't know how to bundle multiple commits into one pull requests so I created a pull request for each change: #910 #911 #912 #913 #914 #915 #916 #917 #918 #919
